.github/workflows/pr-close.yml

```diff
@@ -0,0 +1,29 @@
+name: Cleanup PR preview
+
+# Default to bash
+defaults:
+    run:
+        shell: bash
+
+on:
+    pull_request:
+        types: [closed]
+
+jobs:
+    cleanup_preview:
+        name: Cleanup Preview
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v3
+            - uses: actions/setup-node@v3
+            - uses: Genez-io/genezio-github-action@v1
+              with:
+                  token: ${{ secrets.GENEZIO_TOKEN }}
+            - name: Compute stage
+              run: |
+                  echo "STAGE=z$(echo -n ${{ github.ref_name }} | shasum | head -c 7)" >> $GITHUB_ENV
+                  echo "$STAGE - ${{ github.ref_name }}"
```

Comment:
This `echo` doesn't seem useful.

For debugging purposes the first echo is enough because GitHub expands `${{ github.ref_name }}` in the logs anyway:
<img width="680" alt="image" src="https://github.com/Genez-io/genezio-documentation/assets/26139486/5c4079a8-1a18-4ff1-b173-1e128455dee8">